src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_db.erl

```diff
@@ -182,3 +183,18 @@ fetch_group(#oc_chef_authz_context{reqid = ReqId, server_api_version = ApiVersio
         {error, _} = Error ->
             Error
     end.
+
+find_org_actors_by_name(OrgId, ActorNames) ->
+    case sqerl:select(find_org_actors_by_name, [OrgId, ActorNames]) of
+        {ok, L} when is_list(L) ->
+            % WIP - change this to a record.
```

Comment:
There's another 1 or 2 TODOs floating around that's going to wrapped up in a separate PR later today - when putting that one together I'll make sure they're all cleaned when I submit it.